src/index.js

```diff
@@ -1,21 +1,27 @@
 import React from "react";
 import ReactDOM from "react-dom";
+// <ms_docref_import_css>
 import "bootstrap/dist/css/bootstrap.min.css";
+// </ms_docref_import_css>
 import "./styles/index.css";
 import App from "./App.jsx";
```

Comment:
Why is this not included?

Reply:
Hello @tnorling, In the docs https://review.docs.microsoft.com/en-us/azure/active-directory/develop/tutorial-v2-react?branch=pr-en-us-162748 we need

```
import "bootstrap/dist/css/bootstrap.min.css";
import { PublicClientApplication } from "@azure/msal-browser";
import { MsalProvider } from "@azure/msal-react";
import { msalConfig } from "./authConfig";
```

But these lines don't follow each other in the source code file: https://github.com/Azure-Samples/ms-identity-javascript-react-spa/blob/main/src/index.js

I am proposing we refactor the source code sample to match what is in the docs.